src/parsers/power_system_table_data.jl

```diff
@@ -550,7 +550,8 @@ function loadzone_csv_parser!(sys::System, data::PowerSystemTableData)
 
         buses = get_buses(sys, bus_numbers)
         name = string(zone)
-        load_zones = LoadZones(zone, name, buses, sum(active_powers), sum(reactive_powers))
+        zoneid = typeof(zone) <: Number ? zone : count
```

Comment:
Readability improvement:

```suggestion
        zoneid = zone isa Number ? zone : count
```